src/nft/components/explore/Table.tsx

```diff
@@ -145,93 +146,91 @@ export function Table<D extends Record<string, unknown>>({
   }
 
   return (
-    <GlowEffect>
```

Comment:
removed glow effect here so need to be sure we arent regressing anything due to merge weirdness on the nesting